tox.ini

```diff
@@ -1,4 +1,5 @@
 [tox]
+isolated_build = true
```

Comment:
It seems like from tox v4, [isolated build environments are always used](https://tox.wiki/en/4.15.0/upgrading.html#isolated-environment-on-by-default), so do you think there's any value in adding this field..? 🤔

Reply:
Alright 👍 [We're on 4.15](https://github.com/jazzband/django-simple-history/blob/9e1b5d263f765be7194036da96d7412880456b0a/requirements/tox.txt#L2), so I'll remove it 🙂